### ISSUE TITLE
Facets should be lists instead of headers

### ISSUE THREAD

**Issue by caseyedavis12** [WGBH/AAPB2#1086]
The facets should be a list instead of multiple headers. Related to #1085

**Comment by jasoncorum** [comment]
It appears that the facet lists are already made up of <li> items. Let's review in our next check-in.

**Comment by jasoncorum** [comment]
I actually see where this is now. About to push a code update.
